connectors/services/job_scheduling.py

```diff
@@ -176,7 +176,7 @@ async def _should_schedule_scheduled_sync(job_type):
                 return False
 
             job_type_value = job_type.value
-            now = datetime.utcnow()
+            now = datetime.now(timezone.utc)
```

Comment:
Should we move all the datetime related functions to `utils` or a new `datetime` module?

Reply:
Eventually yes, should be something like `utils/datetime.py`. I'm not planning it in this PR though, I already have a lot of problems with this change alone.